Assets/Scripts/Utilities/DayCycleManager.cs

```diff
@@ -4,55 +4,61 @@
 
 public class DayCycleManager : SingletonBehaviour<DayCycleManager> 
 {
+	public delegate void EndOfDayCallback();
+
 	public static float dayCycleLength { get { return instance._dayCycleLength; } }
 
+	[ReadOnly( "Current Time" ), Tooltip( "0 is midnight. 1/2 of Day Cycle Length is noon." ), SerializeField]
+	float _currentTime = 0f;
+
 	[Tooltip( "The length (in seconds) of a day" )]
-	[SerializeField] float _dayCycleLength = 60.0f;
+	[SerializeField] float _dayCycleLength = 300.0f;
```

Comment:
You shouldn't be worried about updating this when you change the value in the inspector. Just keep it to a sane default value.